### ISSUE TITLE
Fullscreen Portrait Only on Vertical Videos

### ISSUE THREAD

**Issue by Roman19S** [futo-org/grayjay-android#1305]
### Use Cases

People like me only want portrait fullscreen on vertical videos because it fulls the screen for vertical videos, but hurts the experience on horizontal videos by replacing useful info with black bars.

### Proposal

Make the Fullscreen Portrait toggle a drop-down menu that has "off", "on", and "only for vertical videos". The first two options can act as the currently do, and the third should only do portrait fullscreen on videos that have a vertical resolution that is greater than its horizontal resolution.

**Comment by cosmicrabbit452** [comment]
Yes please! Watching pinball tutorials is basically impossible on Grayjay without zooming and cutting out half the video, or straining your eyed with the screen at your nose
